source/game/field/obj/ObjDrivableHolder.hh

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <Common.hh>
+#include "game/field/obj/GeoObject.hh"
+
+namespace Field {
+
+class ObjDrivableHolder {
+public:
+    static ObjDrivableHolder* CreateInstance();
```

Comment:
`static` members come after non-static members. Move to end of `public` section.